backend/services/prisonerProfileService.ts

```diff
@@ -88,9 +94,30 @@ export default ({
         pathfinderApi.getPathfinderDetails(systemContext, offenderNo),
         socApi.getSocDetails(systemContext, offenderNo, socEnabled),
         allocationManagerApi.getPomByOffenderNo(context, offenderNo),
+        curiousApi.getLearnerNeurodivergence(systemContext, offenderNo),
       ].map((apiCall) => logErrorAndContinue(apiCall))
     )
 
+    let hasDivergenceSupport = false
+    const hasSupportNeed = !(
```

Comment:
This is obscuring the other code in the function - it needs to be moved to it's own function